<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dial_ThorDaq</class>
 <widget class="QDialog" name="Dial_ThorDaq">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>263</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Thor Daq</string>
  </property>
  <widget class="QPushButton" name="pushButton_Do">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Do It</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Close">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_Msg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>361</width>
     <height>79</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Actived">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="text">
    <string>Actived</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
